rules/choosing-authentication/rule.md

```diff
@@ -31,80 +31,64 @@ Choosing the right [authentication and authorization](https://www.youtube.com/wa
 
 ### Start with the Questions
 
-1. **Scope** - Is it an enterprise application for internal users or a consumer application for external use?
-2. **Social** - Do you need to support   
-      - [OAuth2](https://oauth.net/2) or 
-      - [OIDC](https://openid.net/connect)?
-3. **MFA** - Do you need to support   
-      - [MFA](https://en.wikipedia.org/wiki/Multi-factor_authentication)?
-4. **Scope** - Do you need to share the identity across multiple applications?
-5. **Volume** - Do you have an estimate for how many users you need to support?
-6. **Complexity** - do you need custom logic to run as part of your authentication process?
+* **Scope** - Is it an enterprise application for internal users or a consumer application for external use?
+* **Scope** - Do you need to share the identity across multiple applications?
+* **Volume** - Do you have an estimate for how many users you need to support?
+* **Complexity** - do you need custom logic to run as part of your authentication process?
```

Comment:
I understand whats being asked here but most people wont know what is going to end up being custom logic and personally its the one thing that will trip people up. If you say no here and turns out you do, you are in trouble, so I think its worth while teasing this out a bit.

Complexity - Do you need additional logic run in your authentication process? some examples might be: 

calling out to other systems to add additional claims,
....

I would steer clear of "Custom" everywhere, IMO decision makers hate it.

Reply:
Agree with @drwharris.  I would like to see some real world examples as to what 'custom logic' means.

Reply:
Good point. Done.